### PR TITLE
Update to CI v4.4.1

### PR DESCRIPTION
**Pull request by kenjis**
- update by tatter/patch
- update coding style
- ~~add `<env name="app.forceGlobalSecureRequests" value="false"/>` in phpunit.xml.dist~~

### CODE REVIEW

**Comment by kenjis** [comment]
Why does the test fail? It passes on my local.

```
There was 1 failure:

1) BasicPagesTest::testCanViewHome
Failed asserting that 307 is identical to 200.

/home/runner/work/website/website/vendor/codeigniter4/framework/system/Test/TestResponse.php:149
/home/runner/work/website/website/tests/feature/BasicPagesTest.php:19
```
https://github.com/codeigniter4projects/website/actions/runs/6033515315/job/16370306869?pr=393

File: phpunit.xml.dist [hunk 1]
```diff
@@ -74,6 +74,8 @@
 
 	<php>
 		<env name="XDEBUG_MODE" value="coverage"/>
+		<env name="app.forceGlobalSecureRequests" value="false"/>
```
Comment (kenjis) [inline phpunit.xml.dist:77]: This code makes the tests pass.
So why did the tests pass without this code in the past?

**Review by Arhell — APPROVED** [review]

**Comment by kenjis** [comment]
Rebased to resolve conflict.

**Review by lonnieezell — APPROVED** [review]

File: phpunit.xml.dist (hunk 1 above)
Comment (kenjis) [reply phpunit.xml.dist:77]: I found a bug in CI v4.0.0 FeatureTestTrait.

Comment (MGatner) [reply phpunit.xml.dist:77]: Nice sleuthing! We can proceed here with test workaround or wait for 4.4.1

Comment (kenjis) [reply phpunit.xml.dist]: Dropped the commit.

**Comment by kenjis** [comment]
Updated to v4.4.1.

**Review by lonnieezell — APPROVED** [review]

**Review by MGatner — APPROVED** [review]

**Comment by MGatner** [comment]
Still good!

**Pull request merged**